package.json

```diff
@@ -37,20 +68,5 @@
       "buildResources": "dist",
       "output": "release"
     }
-  },
-  "dependencies": {
-    "axios": "^0.24.0",
-    "cheerio": "^1.0.0-rc.10",
-    "copyfiles": "^2.4.1",
-    "jquery": "^3.6.0",
-    "typescript": "^4.2.4"
-  },
-  "devDependencies": {
-    "@types/jquery": "^3.5.6",
-    "electron": "^15.3.1",
-    "electron-builder": "^22.13.1",
-    "ts-loader": "^9.2.2",
-    "webpack": "^5.37.1",
-    "webpack-cli": "^4.7.0"
   }
-}
\ No newline at end of file
+}
```

Comment:
I sorted your package.json alphabetically, and I added commit message linting to allow for automatic changelogs in the future

Reply:
I'm not seeing a reason why it needs to be alphabetic. For husky, can you explain what that is? I'm unfamiliar with it.

Reply:
As for alphabetic sorting, vscode forced my hand

Reply:
Oh cool. Husky will be helpful then